### DISCUSSION TITLE
I moved osu!lazer folder, then moved it back, and lost all data.

### DISCUSSION THREAD

**Discussion by Yomiia** [ppy/osu#23192] [Q&A] [answered]
My osu!lazer folder was in B:\Games, I moved it to B:\Games\Rhythm. When I opened lazer, all the beatmaps, scores, skins and my profile were gone, and a message popped up saying to move the osu!lazer folder back where it was. So I moved it back to B:\Games, but the save data is still gone. What happened? Where are the beatmaps and scores stocked on my PC, and how can I tell lazer where they are?

**Comment by Theighlin** [comment, marked as answer]
Open the game, in settings look for a "move osu folder" button, and point it to the new location, if you pointit to an already existing lazer data folder, it should pick up on that.
